public/style.css

```diff
@@ -2,24 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-@font-face {
-  font-family: 'Open Sans';
-  font-style: normal;
-  font-weight: 300;
-  src: local('Open Sans Light'), local('OpenSans-Light'), url('OpenSans-Light.woff') format('woff');
-}
-
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font: 400 16px/1.42857143 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Could we maybe keep `Open Sans` in the list (without providing it as a downloadable `woff` file) for those people lucky enough to have it installed locally already?